README.md

```diff
@@ -47,6 +47,13 @@ let g:goimports = 0
     unlet! g:goimports_simplify
     ```
 
+* `g:goimports_local` - use `-local` option when running `goimports`.
```

Comment:
Thank you. Please append this into here.

> This is useful to import  closed-source packages. (comma separated list)

Reply:
updated. Thank you for reviewing.